# Available commands to use the Fisher Scientific PPS4102 Top Pan balance 
int32 HEATON = 0
int32 HEATOFF = 1
int32 STIRON = 2
int32 STIROFF = 3
int32 SETSTIR = 4
int32 SETHEAT = 5
int32 STIRAT = 6
int32 HEATAT = 7
int32 ALLOFF = 8

int32 ika_command
int32 ika_param
